pkg/mkverification-raw-efi/verify

```diff
@@ -374,24 +374,27 @@ do
 done
 
 #Testing qemu and passthrough
-qemu_exec=/usr/bin/qemu-system-$(uname -m)
-machine=$($qemu_exec -machine help | awk '{if (NR==2) print $1}')
-$qemu_exec -m 1024 -smp 2 -display none -serial mon:stdio -global ICH9-LPC.noreboot=false \
-   -watchdog-action reset -rtc base=utc,clock=rt -machine "$machine" -cpu SandyBridge \
-   -drive file=/ubuntu-22.04-minimal-cloudimg-amd64.img,format=qcow2 &
-
-# shellcheck disable=SC2181
-if [ "$?" -eq 0 ]; then
-   kill %1
-   echo "start VM success" > "$REPORT/guest-checks.log"
-   echo "Start of an edge application successful" >> "$REPORT/summary.log";
-else
-   echo "start VM failed" > "$REPORT/guest-checks.log"
-   echo "Start of an edge application failed" >> "$REPORT/summary.log";
-fi
+#qemu_exec=/usr/bin/qemu-system-$(uname -m)
```

Comment:
This commenting out of a large number of lines isn't very nice.
Better to use an if false; then" block with a comment indicating why and what it would take to add it back in.

Reply:
OK, I will do that.

Reply:
I used 

```
: ' <block> ' 
```

Which is a multi-line comment for scripts.

Reply:
Which IMHO  is unreadable when there is code in there. If you wanted to write some english prose it would be readable.